shared/message/output/Say.proto

```diff
@@ -21,10 +21,12 @@ syntax = "proto3";
 package message.output;
 
 /**
- * TODO document
- *
- * @author Trent Houliston
+ * Is a string for the robot to say.
  */
 message Say {
+
+    /**
+     * The string to say.
+     */
```

Comment:
When documenting fields just use a triple slash rather than having a full block, you've done this throughout and it greatly increases the number of lines which makes it harder to read
```suggestion
    /// The string to say.
```